cli/cmd/run.go

```diff
@@ -87,7 +90,20 @@ func buildRunStackCommand(key string, stack config.Stack) *cobra.Command {
 					"stack": stack.Name,
 				}
 				s := serviceManager.BuildFromConfig(srv)
-				serviceManager.Run(s)
+
+				if k == "kibana" {
+					s.SetWaitFor(wait.ForLog("http server running"))
+				}
+
+				if k == "elasticsearch" {
+					serviceManager.Run(s)
+				} else {
+					servicesToRun[k] = s
+				}
```

Comment:
Why adding special cases for kibana and elasticsearch?

Reply:
To implement the startup order, and following the BigPicture diagram (https://docs.google.com/drawings/d/1MJ-haufU06JnUHYxYDwP-EH-xBpyMDrpDn2UEdS2WgA/edit) we must do those checks.

If you look at the picture, and create a tree-graph, you'll get that elasticsearch is the root node, so the first one to be started. Next in the line would be the APM server...

Reply:
It'd be good to have some kind of dependencies mechanism to avoid these special cases (like `depends_on` in docker compose).

Reply:
Yeah, we do not have it in place now. It'd be something to work on, or wait for the potential move to compose...